src/common/putslave.cpp

```diff
@@ -369,29 +367,6 @@ int putslave_do_cmd_a_start(uartslave_t *pslave) {
 }
 #endif
 
-int putslave_do_cmd_a_stop(uartslave_t *pslave) {
-    if (marlin_server_processing()) {
-        marlin_server_stop_processing();
-        osThreadSuspend(displayTaskHandle);
-#ifdef BUDDY_ENABLE_ETHERNET
-        osThreadSuspend(webServerTaskHandle);
-#endif //BUDDY_ENABLE_ETHERNET
-#if HAS_GUI
-        HAL_SPI_MspDeInit(&hspi2);
-#endif
-        NVIC_DisableIRQ(TIM7_IRQn);
-        hwio_pwm_set_val(HWIO_PWM_HEATER_BED, 0);
-        hwio_pwm_set_val(HWIO_PWM_HEATER_0, 0);
-        //SCK - PB10
-        gpio_init(TPB10, GPIO_MODE_OUTPUT_PP, GPIO_NOPULL, GPIO_SPEED_FREQ_HIGH);
-        //MISO - PC2
-        gpio_init(TPC2, GPIO_MODE_INPUT, GPIO_PULLUP, GPIO_SPEED_FREQ_VERY_HIGH);
-        //MOSI - PC3
-        gpio_init(TPC3, GPIO_MODE_OUTPUT_PP, GPIO_NOPULL, GPIO_SPEED_FREQ_HIGH);
-    }
-    return UARTSLAVE_OK;
-}
-
```

Comment:
Putslave "stop" can not be removed, as it is used for retesting already tested and locked boards.